File: app/src/main/java/me/blog/korn123/easyphotomap/helper/PhotoMapMigration.java
package me.blog.korn123.easyphotomap.helper;

import io.realm.DynamicRealm;
import io.realm.RealmMigration;
import io.realm.RealmObjectSchema;
import io.realm.RealmSchema;

/**
 * Created by CHO HANJOONG on 2017-09-12.
 */

public class PhotoMapMigration implements RealmMigration {

    @Override
    public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
        RealmSchema schema = realm.getSchema();
        if (oldVersion == 1) {
            RealmObjectSchema diarySchema = schema.get("PhotoMapItem");
            oldVersion++;
        }
    }

}

File: app/src/main/java/me/blog/korn123/easyphotomap/helper/PhotoMapMigration.kt
package me.blog.korn123.easyphotomap.helper

import io.realm.DynamicRealm
import io.realm.RealmMigration

/**
 * Created by CHO HANJOONG on 2017-09-12.
 */

class PhotoMapMigration : RealmMigration {

    override fun migrate(realm: DynamicRealm, oldVersion: Long, newVersion: Long) {
        var currentVersion = oldVersion as Int
        val schema = realm.schema
        if (currentVersion == 1) {
            val diarySchema = schema.get("PhotoMapItem")
            currentVersion++
        }
    }

}
